src/infrastructure/figma/figma-service.ts

```diff
@@ -201,6 +201,11 @@ export class FigmaService {
 					{ errors: response.errors },
 					'Failed to create Figma dev resources.',
 				);
+
+				const { error } = response.errors[0];
```

Comment:
**question:** If I am not missing anything, based on the existing code, it seems `tryCreateDevResourceForJiraIssue` does not throw when `response.errors` is not empty. This being the case, could we check why does `POST /entities/associateEntity` return `HTTP 500` when dev resource creation fails with an error?